src/modules/CVs/ui/cvs-list.tsx

```diff
@@ -0,0 +1,66 @@
+import { useNavigate } from "react-router-dom";
+import { t } from "i18next";
+import ListTemplate from "@/components/list-lemplate/list-template";
+import { Action } from "@/components/list-lemplate/actions-menu";
+import { ColumnConfig } from "@/components/list-lemplate/table-template";
+import { useConfirm } from "@/components/confirm-dialog/";
+import { useCvCreate, useCvDelete, useCvsQuery } from "../api";
+import { useCvDialog } from "./cvs-dialog";
+import { CvTransformed, mapCvDataToTable } from "@/helpers/convert/maps";
+
+const columnConfigs: ColumnConfig<CvTransformed>[] = [
+    { name: "name", isSorted: true },
+    { name: "description", isSorted: false },
+    { name: "employee", isSorted: true },
+];
+
+const CvsList = () => {
+    const { cvs, loading } = useCvsQuery();
+
+    const navigate = useNavigate();
+
+    const [openConfirm] = useConfirm();
+    const [openCvDialog] = useCvDialog();
+    const [createCv] = useCvCreate();
+    const [deleteCv] = useCvDelete();
+
+    const menuProps: Action = {
+        onDelete: (id: string) =>
+            openConfirm({
+                title: t("delete confirmation"),
+                onConfirm: () => void deleteCv({ variables: { cv: { cvId: id } } }),
+            }),
+
+        onUpdate: (id: string) => navigate(`/routes/cvs/${id}`),
```

Comment:
can replace this route template string with an existing helper function (routes.cvs.cvById(id) (or routes.cvs.details(id), since that would be the default tab anyway))

Reply:
change